test/crypto_subtle_test.dart

```diff
@@ -261,7 +275,7 @@ void main() {
               false,
             )
             .having(
-              (key) => key.privateKey.usages,
+              (key) => key.privateKey.usages.toDartList,
```

Comment:
Can we do the `.usages.toDartList` as a separate PR.

It seems to me like we can get this passing all existing tests. That should make it easy to land, and it'll reduce the size of this PR.

Reply:
I made a separate PR.
https://github.com/google/webcrypto.dart/pull/181